ansible_collections/arista/avd/examples/dual-dc-l3ls/README.md

```diff
@@ -0,0 +1,609 @@
+| DC1                                                 | 10.255.1.64/27              |
+| DC2                                                 | 10.255.129.64/27            |
+| **MLAG iBGP Peering (interface vlan 4093)**         | **(Leaf switches)**         |
+| DC1                                                 | 10.255.1.96/27              |
+| DC2                                                 | 10.255.129.96/27            |
+
+### BGP design
+
+New BGP connections are established from dc1-leaf2a and dc1-leaf2b to dc2-leaf2a and dc2-leaf2b respectively. In addition, all BGP learned routes are redistributed into the underlay. This allows for an easier troubleshooting, as all router IDs (Loopback0 IP addresses) and VTEPs (Loopback1 IP addresses) are reachable at a fabric level.
+
+From the overlay perspective, each new leaf sees its peer in the twin DC as a new Route Server and will advertise all EVPN learned routes as they were directly connected locally, making all changes inside its DC transparent to the twin DC.
+
+=== "Underlay"
+
+    ![Figure: Arista Underlay BGP Design](images/bgp-underlay.svg)
+
+=== "Overlay"
+
+    ![Figure: Arista Underlay BGP Design](images/bgp-overlay.svg)
+
+### Basic EOS config
+
+As discussed in the single DC example, basic connectivity between the Ansible host and the switches must be established before Ansible can be used to push configurations. Remember, you must configure the following on all switches:
+
+- A hostname configured purely for ease of understanding.
+- An IP enabled interface - in this example the dedicated out-of-band management interface is used.
+- A username and password with the proper access privileges.
+
+!!! note
+    The folder `dual-dc-l3ls/switch-basic-configurations/` contains a file per device for the initial configurations.
+
+## Ansible inventory, group vars and naming scheme
+
+The following drawing shows a graphic overview of the Ansible inventory, group variables, and naming scheme used in this example:
+
+![Figure: Ansible inventory and vars](images/ansible-groups.svg)
+
+!!! note
+    The two servers on DC1 (`dc1-leaf1-server1` and `dc1-leaf2-server1`) and the two servers on DC2 (`dc2-leaf1-server1` and `dc2-leaf2-server1`) at the bottom are **not** configured by AVD, but the switch ports used to connect to the servers are.
+
+The same pattern used in the single DC example is followed:
+
+- Group names use uppercase and underscore
+- All hostnames use lowercase and dashes
+
+The drawing also shows the relationships between groups and their children. Be aware that all declarations on a higher level are inherited by chidren automatically.
+
+### Content of the inventory.yml file
+
+This section describes the entire `ansible-avd-examples/dual-dc-l3ls/inventory.yml` file used to represent the above topology.
+
+In this example, we will consider that no DNS is available and define the IPs that the Ansible host has to reach per device.
```

Comment:
I can see from a virtual example point of view that having all management IPs in a single contiguous subnet is easier, but for a more real-world scenario, wouldn't it make sense to use dedicated / separate subnets?

Reply:
done